app/views/request/_sidebar.html.erb

```diff
@@ -2,17 +2,6 @@
   <div class="new-request-cta">
     <%= render :partial => "general/new_request" %>
   </div>
-  <div id="track-request" class="track-request">
-    <h2><%= _('Follow this request') %></h2>
-
-    <p>
-      <%= n_("There is {{count}} person following this request",
-             "There are {{count}} people following this request",
-             @follower_count,
-             :count => @follower_count) %>
-    </p>
```

Comment:
We could reuse that here, space is a bit more limited but I don't think it will be a huge issue